waku/incentivization/common.nim

```diff
@@ -0,0 +1,13 @@
+import std/options
+
+import waku/incentivization/rpc
+
+proc genEligibilityStatus*(isEligible: bool): EligibilityStatus = 
```

Comment:
This seems to me to be interesting only for tests? In other words, this should form part of the test files as a helper function.

Reply:
My idea was, on the contrary, that this function (renamed to `new` in later commits) would be useful for future code that would use our eligibility logic to convert a boolean eligibility value into `EligibilityStatus` that will be attached to the response. Something like this:

```
let proof = ... # extract eligibility proof (not necessarily a txid) from the request
let isValid = isEligible(proof, ...) # check proof eligibility
let eligibilityStatus = new(EligibilityStatus, isValid)
let response = LightpushResponse(eligibilityStatus, ...) # generate response
return response # (send the response back to the client)
```

Does it make sense?